test/renderer_test.exs

```diff
@@ -91,4 +91,18 @@ defmodule RendererTest do
   test "CRLF line endings are converted to LF" do
     assert render("h1\r\n\th2\r\n\t\th3 Hi\r\n") == "<h1><h2><h3>Hi</h3></h2></h1>"
   end
+
+  test "CRLF line endings corner case" do
+    example_unix = """
+    html
+      head
+        meta
+
+      body
+    """
+    example_windows = example_unix |> String.replace("\n", "\r\n")
```

Comment:
It doesn't do the same: it fails on v1.0.0 with an error. Independently of the OS, elixir normalizes the line endings in multiline strings. Line endings will always be "\n", both in windows and linux.

By replacing the linux line endings with windows line endings you mimic the effect of running slime on windows files.

I can compare with the expected output, of course, if you want.